ccd-definition/CaseField.json

```diff
@@ -73,67 +73,43 @@
     "FieldType": "Label",
     "SecurityClassification": "Public"
   },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "Shared_Storage_DRAFTType",
-    "ID": "documents_socialWorkChronology_question",
-    "Label": "# 1. Social work chronology",
-    "FieldType": "Label",
-    "SecurityClassification": "Public"
-  },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "Shared_Storage_DRAFTType",
     "ID": "documents_socialWorkChronology_document",
-    "Label": "Social work chronology",
+    "Label": "# 1. Social work chronology",
     "FieldType": "UploadDocument",
     "SecurityClassification": "Public"
   },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "Shared_Storage_DRAFTType",
-    "ID": "documents_socialWorkStatement_question",
-    "Label": "# 2. Social work statement and genogram",
-    "FieldType": "Label",
-    "SecurityClassification": "Public"
-  },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "Shared_Storage_DRAFTType",
     "ID": "documents_socialWorkStatement_document",
-    "Label": "Social work statement and genogram",
+    "Label": "# 2. Social work statement and genogram",
     "FieldType": "UploadDocument",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "Shared_Storage_DRAFTType",
-    "ID": "documents_socialWorkAssessment_question",
+    "ID": "documents_socialWorkAssessment_document",
     "Label": "# 3. Social work assessment",
-    "FieldType": "Label",
+    "FieldType": "UploadDocument",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "Shared_Storage_DRAFTType",
     "ID": "documents_socialWorkAssessment_document",
-    "Label": "Social work assessment",
+    "Label": "# 3. Social work assessment",
```

Comment:
is this not the same as the one above now?

Reply:
yep, remove one of the objects and I think we're good

Reply:
and youll need to remove it from case event to fields

Reply:
all instances of documents_socialWorkChronology_question removed now as CCD was looking for an instance of it that was not there due to change. Fixed and here are the screenshots of how it looks now.
<img width="651" alt="screenshot 2018-12-07 at 15 17 57" src="https://user-images.githubusercontent.com/42037338/49655779-7ea7c380-fa33-11e8-85e4-ffae21a8a081.png">
<img width="645" alt="screenshot 2018-12-07 at 15 18 03" src="https://user-images.githubusercontent.com/42037338/49655780-7ea7c380-fa33-11e8-92a4-7506cbd3400b.png">
<img width="692" alt="screenshot 2018-12-07 at 15 18 08" src="https://user-images.githubusercontent.com/42037338/49655781-7ea7c380-fa33-11e8-8002-5747762ecdc6.png">

Reply:
youve got the hash in there, its not applied the markup properly